src/Controller/ControllerManager.php

```diff
@@ -73,10 +74,10 @@ public function injectControllerDependencies($controller, ServiceLocatorInterfac
             // is why the shared EM injection needs to happen; the conditional
             // will always pass.
             $events = $controller->getEventManager();
-            if (!$events instanceof EventManagerInterface) {
+            if (! $events instanceof EventManagerInterface
```

Comment:
Urgh, wait, the interface defines no getters... https://github.com/zendframework/zend-eventmanager/blob/a6c747fe59658fb62feb23613d7ea22146b6b611/src/EventManagerAwareInterface.php#L15-L24